docs/source/markdown/podman-remote.1.md

```diff
@@ -132,14 +132,3 @@ the exit codes follow the `chroot` standard, see below:
 | [podman-unpause(1)](podman-unpause.1.md)         | Unpause one or more containers.                                             |
 | [podman-version(1)](podman-version.1.md)         | Display the Podman version information.                                     |
 | [podman-volume(1)](podman-volume.1.md)           | Manage Volumes.                                                             |
-
-## FILES
```

Comment:
Could you replace this with containers.conf

Reply:
We are going to need the containers.conf man page or a slimmed down one, along with the containers-storage.conf and containers-image.conf man pages.

@vrothberg Back in the day we could not maintain these in vendor directory, is this still true?